Content.Server/_Goobstation/Magic/ImmovableVoidRodSystem.cs

```diff
@@ -57,9 +57,9 @@ private void OnCollide(Entity<ImmovableVoidRodComponent> ent, ref StartCollideEv
         TryComp<TagComponent>(args.OtherEntity, out var tag);
         var tags = tag?.Tags ?? new();
 
-        if (tags.Contains("Wall") && Prototype(args.OtherEntity) != null && Prototype(args.OtherEntity)!.ID != "WallSnowCobblebrick")
+        if (tags.Contains("Wall") && Prototype(args.OtherEntity) != null && Prototype(args.OtherEntity)!.ID != ent.Comp.SnowWallPrototype)
```

Comment:
`Prototype(args.OtherEntity)` should probably be pattern matched rather than null forgiving at `Prototype(args.OtherEntity)!.ID`